trl/trainer/sft_trainer.py

```diff
@@ -224,7 +224,8 @@ class SFTTrainer(Trainer):
         peft_config ([`~peft.PeftConfig`], *optional*, defaults to `None`):
             PEFT configuration used to wrap the model. If `None`, the model is not wrapped.
         formatting_func (`Optional[Callable]`):
-            Formatting function applied to the dataset before tokenization.
+            Formatting function applied to the dataset before tokenization. Applying the formatting function explicitly converts
+            the dataset into a language modeling format.
```

Comment:
```suggestion
            the dataset into a language modeling type.
```

"format" is for conversational/standard